src/Robo/Plugin/Commands/Drupal7DevelopmentModeCommands.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace ChqRobo\Robo\Plugin\Commands;
+
+use Robo\Exception\TaskException;
+use Robo\Result;
+
+/**
+ * Robo commands related to changing development modes for Drupal 7.
+ */
+class Drupal7DevelopmentModeCommands extends DevelopmentModeBaseCommands
+{
+    /**
+     * Refreshes a Drupal 7 development environment.
+     *
+     * Completely refreshes a development environment including running 'composer install', starting Lando,
+     * downloading a database dump, importing it, running deployment commands, disabling front-end caches,
+     * and providing a login link.
+     *
+     * @param string $siteName
+     *   The Drupal site name.
+     *
+     * @return \Robo\Result
+     *   The result of the set of tasks.
+     */
+    public function devRefreshLegacy(string $siteName = 'default'): Result
```

Comment:
Maybe this should be D7 specific?

```suggestion
    public function devRefreshDrupal7(string $siteName = 'default'): Result
```